### DISCUSSION TITLE
why double comma ?

### DISCUSSION THREAD

**Discussion by sourcevault** [gerardroche/gerardroche.github.io#50] [General]
In this blog post:

https://blog.gerardroche.com/2023/05/18/neovintageous-reveal-in-side-bar/

At the end you have this binding:

```js
{
    "keys": [",", ","],
    "command": "focus_group",
    "args": {"group": 0},
    "context": [
        {"key": "control", "operand": "sidebar_tree"}
    ]
},
```

Is there a particular reason why it is double comma instead of single ?

**Comment by gerardroche** [comment]
Because my leader key is a comma:

```
let mapleader=,
```

The default leader key is a backslash, but a lot vim users use comma.

I map the reveal in sidebar to two commas `,,` i.e. the leader key x2 

```
let mapleader=,
nnoremap <leader><leader> :Neovintageous action=reveal_side_bar<CR>
```

So, when I do `<leader>` x2 the file is revealed in the side bar and focus is put on the sidebar. Now you can do `ctrl+1` or `Enter` to put the focus back on the view, but I like the same mapping to go back to the view: two commas `,,` or two leader keys.

I do the same things with a toggle side bar mapping: https://blog.gerardroche.com/2023/05/15/neovintageous-toggle-sidebar/
